src/crudeditor-lib/rootReducer.js

```diff
@@ -17,11 +17,13 @@ import {
 
 export default modelDefinition => combineReducers({
   common: common(modelDefinition),
-  views: combineReducers({
-    [VIEW_SEARCH]: search(modelDefinition),
-    [VIEW_CREATE]: create(modelDefinition),
-    [VIEW_EDIT]: edit(modelDefinition),
-    [VIEW_SHOW]: show(modelDefinition),
-    [VIEW_ERROR]: error(modelDefinition)
-  })
+  views: combineReducers(
+    (ops => ({
```

Comment:
Build the object directly instead of indirectly via a function.

Reply:
Done.